restconnector/RequestHandler.cpp

```diff
@@ -83,7 +95,6 @@ void RequestHandler::processGET_( const ::zeq::Event& event )
 
     while( blocked_ )
     {
-        while( subscriber_->receive( 0 ) ){}
```

Comment:
You're right, what I wrote is nonsense. The right code is:

    while( blocked_ )
       subscriber_->receive();